src/java/org/apache/cassandra/service/accord/AccordKeyspace.java

```diff
@@ -156,12 +174,33 @@ ImmutableSortedMap<Timestamp, ByteBuffer> getValues(CommandsForKey cfk)
         }
     }
 
+    private enum TokenType
+    {
+        Murmur3((byte) 1),
+        ;
+
+        private final byte value;
```

Comment:
my thinking is that a `murmur3` token can't compare with a `bytes order` token, so by pulling out the `type` as the bytes prefix, we know that only `murmur3` tokens will be seen during the query if the range is for a `murmur3` token range